spec.md

```diff
@@ -925,11 +1050,44 @@ networks:
         - subnet: "2001:3984:3989::/64"
 ```
 
+### oom_kill_disable
+ 
+If `oom_kill_disable` is set Compose implementation MUST configure the platform so it won't kill the container in case 
+of memory starvation.
```

Comment:
There's been some discussion about this option overall; it's _really_ dangerous (process can never be killed, which means the host will be killed before this container). I'd be very cautious bringing this back.

`oom_score_adj` would already achieve this in 99% of the situations. (`-1000` is "unkillable")

Reply:
again, we bring it back to the spec because it existed in v2 and we want to give implementation a chance to parse any compose file and manage such attribute the way they want (can just ignore with a warning). We can discuss about deprecating any dangerous feature in another issue